cuckoo/cuckoo_result.py

```diff
@@ -1727,6 +1723,73 @@ def _remove_safe_roots(process_tree: List[Dict[str, Any]], process_tree_hashes:
             process_tree.pop(index - num_removed)
             num_removed += 1
 
+def _create_hashed_leaf(parent: str, node: Dict[str, Any], hashes: List[str]) -> None:
+    # Extract the two pieces of info needed from the node
```

Comment:
since `_create_hashed_leaf` is almost the exact same code as the `_create_hashed_node` method, can we combine them to avoid code duplication